driver-mapping/src/test/java/com/datastax/driver/mapping/MapperCaseSensitivityTest.java

```diff
@@ -40,7 +40,7 @@
     static class User {
 
         @PartitionKey
-        @Column(caseSensitive = true)
+        @Column(name = "userId", caseSensitive = true)
```

Comment:
Added note about that to upgrade guide in 3b86cef (hope that looks ok, feel free to adjust)